ivadomed/loader/segmentation_pair.py

```diff
@@ -350,17 +350,22 @@ def convert_file_to_nifti(self, filename: str, extension: str, is_gt: bool = Fal
             # Note: All of these are trivially true for JPEG and PNG due to limitations of these formats.
 
             # make grayscale (treats binary as 1-bit grayscale)
-            colorspace_idx = 2 + int(props.is_batch)
+
+
+            # make grayscale (treats binary as 1-bit grayscale)
+            colorspace_idx = 2
             if _img.ndim <= colorspace_idx:  # binary or gray
                 pass  # nothing to do
+            elif _img.shape[colorspace_idx]==2: #gray with alpha channel
```

Comment:
Nit: PEP8 styling (spaces around `==` operator, 2 spaces before comment, space after `#` in comment)

```suggestion
            elif _img.shape[colorspace_idx] == 2:  # gray with alpha channel
```